src/data.js

```diff
@@ -11440,6 +11440,44 @@ module.exports = [
       'Blogger',
     ],
   },
+  {
+    name: 'Tim Smith',
+    description:
+      'I am simply a web developer. I love web development.  I am not currently making money with it, but I write code every day with glee',
+    url: 'https://timsmith.tech/uses',
```

Comment:
This 404s, in order for you to be added to awesome-uses, you'll need a working (& filled out) uses page